### PR TITLE
refactor: (GenAI) Reorganized Controlled Generation Samples (Group A)

### PR DESCRIPTION
**Pull request by Thoughtseize1**
## Description
Reorganized Controlled Generation Samples.
1. Moved each `generate_content` function with region tags into separate files.
2. Rewrote tests

|   Region Tag	| Old file   	| New File  	| 
|---	|---	|---	|
|   generativeaionvertexai_gemini_controlled_generation_response_schema	|   response_schema.py	|   example_01.py	| 
|   generativeaionvertexai_gemini_controlled_generation_response_schema_2	|   response_schema.py	|   example_02.py	|
|   generativeaionvertexai_gemini_controlled_generation_response_schema_3	|   response_schema.py	|   example_03.py	| 
|   generativeaionvertexai_gemini_controlled_generation_response_schema_4	|   response_schema.py	|   example_04.py	| 
|   generativeaionvertexai_gemini_controlled_generation_response_mime_type	|   response_mime_type.py	|   example_05.py	|
|  generativeaionvertexai_gemini_controlled_generation_response_schema_6 	|   response_schema.py	|   example_06.py	|
|  generativeaionvertexai_gemini_controlled_generation_response_schema_7 	|   response_schema.py	|   example_07.py 	|

❗**Files need to be deleted after approval:**
`response_schema.py`
`response_mime_type.py`

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: generative_ai/controlled_generation/example_06.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            "Generate a list of objects in the images.",
+        ],
+        generation_config=GenerationConfig(
+            response_mime_type="application/json", response_schema=response_schema
+        ),
+    )
+
+    print(response.text)
+    # Example response:
+    # [
```
Comment (msampathkumar) [inline generative_ai/controlled_generation/example_06.py:63]: Can you reduce the no.of lines used for model reponse?

```
 [
        [
            {"object": "globe"}, {"object": "tablet"}, {"object": "toy car"},
            {"object": "airplane"}, {"object": "keyboard"}, {"object": "mouse"},
            {"object": "passport"}, {"object": "sunglasses"}, {"object": "money"},
            {"object": "notebook"}, {"object": "pen"}, {"object": "coffee cup"},
        ],
        [
            {"object": "watering can"}, {"object": "plant"}, {"object": "flower pot"},
            {"object": "gloves"}, {"object": "garden tool"},
        ],
    ]
```

File: generative_ai/controlled_generation/example_01.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+    response = model.generate_content(
+        "List a few popular cookie recipes",
+        generation_config=GenerationConfig(
+            response_mime_type="application/json", response_schema=response_schema
+        ),
+    )
+
+    print(response.text)
+    #    Example response:
```
Comment (msampathkumar) [inline generative_ai/controlled_generation/example_01.py:52]:
```suggestion
    # Example response:
```

File: generative_ai/controlled_generation/example_02.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+
+def generate_content() -> str:
+    # [START generativeaionvertexai_gemini_controlled_generation_response_schema_2]
+    import vertexai
+
+    from vertexai.generative_models import GenerationConfig, GenerativeModel
+
+    # TODO(developer): Update and un-comment below line
+    # PROJECT_ID = "your-project-id"
```
Comment (msampathkumar) [inline generative_ai/controlled_generation/example_02.py:26]:
```suggestion
    # project_id = "PROJECT_ID"
```

Please check and update this line for other places too.

File: generative_ai/controlled_generation/example_02.py [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+    response = model.generate_content(
+        prompt,
+        generation_config=GenerationConfig(
+            response_mime_type="application/json", response_schema=response_schema
+        ),
+    )
+
+    print(response.text)
+    #    Example response:
```
[the remark this answers is not in the document]
Comment (Thoughtseize1) [reply generative_ai/controlled_generation/example_02.py:59]: Done

**Review by msampathkumar — APPROVED** [review]
LGTM

**Pull request merged**